lib/features/details/presentation/view/movie_details_page.dart

```diff
@@ -89,7 +116,14 @@ class _MovieDetailsPageState extends State<MovieDetailsPage> {
                   ),
                 ),
                 const SizedBox(height: 20),
-                const SimilarMoviesListView(title: 'Similar Movies'),
+                ValueListenableBuilder(
+                    valueListenable: detailsViewModel.similarMovies,
+                    builder: (context, movie, _) {
+                      return SimilarMoviesListView(
+                        title: 'Similar Movies',
+                        movie: movie,
```

Comment:
This should be named `movies` to indicate that it is a collection and not just one movie